core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedElementNameUtil.java

```diff
@@ -44,6 +44,27 @@ static String findName(Object nameRetrievalTarget, String value) {
         return getName(nameRetrievalTarget);
     }
 
+    /**
+     * Returns the value of the {@link Header} annotation which is specified on the {@code element} if
```

Comment:
Shoud we update Javadoc of the method above?

Reply:
Thanks, fixed. 😉